src/serum-trade-lifecycle.yaml

```diff
@@ -0,0 +1,321 @@
+        answerKeys:
+          - D
+        type: SingleChoice
+        choices:
+          - content: Serum Program Liquidity token
+            key: A
+          - content: Solana Program Liquidity token
+            key: B
+          - content: Serum Program Library token
+            key: C
+          - content: Solana Program Library token
+            key: D
+
+  - uuid: 7435646e-206d-4b76-b402-628746f9f160
+    name: Trade Lifecycle - Placing Orders
+    content: |
+      The trade lifecycle generally consists of four main stages: placing orders, matching orders, consuming events, and settlement.
+      Let's see each in detail:-
+      
+      ## 1. Placing orders:
+      A user funds an intermediary account (their OpenOrders account) from their SPL token account (wallet) and adds an order placement request to the Request Queue.
+      ### Necessary steps required in placing successful orders:-
+        * 1. A user submits a **Place Order** instruction to the DEX program, passing:
+          * Order details: the market size, price, order type, side
+          * Their OpenOrders account on that market
+          * Their SPL token account for the market's base currency (if selling) or quote currency (if buying)
+        * 2. The **DEX program** then:
+          -Transfers funds:
+            * Determines the max funds required for this order (the size if selling, or size * price if buying)
+            * Notes the free balance for the corresponding currency indicated in the OpenOrders account
+            * Transfers the difference between the required amount and free OpenOrders balance from the SPL token account to the Base Currency Vault or Quote Currency Vault.
+            * Increments the OpenOrders account's total balance for the corresponding currency by the amount transferred
+          - Increments and retrieves a sequence number from the Request Queue: this, along with the order's price, determines the new order's ID.
+          - Adds an item to the Request Queue specifying the order details (size, price, order type, side)
+          - Adds an item representing the new order to an array in the OpenOrders account in an available slot
+    
+    stepItems: []
+  - uuid: 6c5397e0-169a-4a28-b077-aa4739b662e9
+    name: Evaluation
+    content: |
+    stepItems: 
+      - uuid: 2fb87226-070d-40c0-94b8-54a4f18f0d94
+        content: Which of the following details in not included in an order placement request?
+        answerKeys:
+          - B
+        type: SingleChoice
+        choices:
+          - content: User's OpenOrders account on the market
+            key: A
+          - content: User's bank account for the market
+            key: B
+          - content: User's SPL token account for the market
+            key: C
+          - content: The market size, price, order type
+            key: D
+
+      - uuid: 5ef22568-2fc3-450d-94e2-2fad26cada2e
+        content: Where does an order placement request gets added?
```

Comment:
The same question was asked in the previous evaluation step also. We should add a different one which relates to the different account types

Reply:
I have updated previous step question